### PR TITLE
Add travis-ci badge

### PR DESCRIPTION
**Pull request by frafra**
Add missing travis-ci badge to README.rst

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,6 +20,10 @@ CKAN: The Open Source Data Portal Software
     :target: https://coveralls.io/github/ckan/ckan?branch=master
     :alt: Coverage Status
 
+.. image:: https://travis-ci.org/ckan/ckan.svg?branch=master
+     :target: https://travis-ci.org/ckan/ckan
+     :alt: Build Status
```
Comment (wardi) [inline README.rst]: Can we tell users that this is a "Docker build status" icon? That's all that travis is doing right now.

Comment (kmbn) [reply README.rst]: That's a good point. @wardi you just mean changing `:alt: Build Status` to `:alt: Docker Build Status`, right?

Comment (frafra) [reply README.rst]: alt changed. Travis CI does not provide an alternative image for that. At the moment Travis CI is working just fine.

**Comment by amercader** [comment]
We are not using Travis any more

**Pull request closed without merging**